packages/discovery-react-components/src/components/SearchFacets/components/FacetsGroups/CollapsibleFacetsGroup.tsx

```diff
@@ -46,6 +50,7 @@ interface CollapsibleFacetsGroupProps {
 
 export const CollapsibleFacetsGroup: FC<CollapsibleFacetsGroupProps> = ({
   facets,
+  showMatchingResults = false,
```

Comment:
do you need a default here? it should always be passed in from the parent component